docs/python-api/query-api.rst

```diff
@@ -1,26 +1,35 @@
 .. _query-api:
 
-Query API
-=========
-Neptune client allows you to fetch data directly form Neptune. This let users make custom analysis of their data.
-
-Project-level
--------------
-On the level of project you can fetch list of :class:`~neptune.experiments.Experiment` objects,
-fetch experiments view as Pandas DataFrame and get all members of the project.
-
-* :meth:`~neptune.projects.Project.get_experiments`
-* :meth:`~neptune.projects.Project.get_leaderboard`
-* :meth:`~neptune.projects.Project.get_members`
-
-Experiment-level
-----------------
-On the level of :class:`~neptune.experiments.Experiment`, you can fetch multiple types of tracked objects, ranging from numeric logs values to tags.
-
-* :meth:`~neptune.experiments.Experiment.get_hardware_utilization`
-* :meth:`~neptune.experiments.Experiment.get_logs`
-* :meth:`~neptune.experiments.Experiment.get_numeric_channels_values`
-* :meth:`~neptune.experiments.Experiment.get_parameters`
-* :meth:`~neptune.experiments.Experiment.get_properties`
-* :meth:`~neptune.experiments.Experiment.get_system_properties`
-* :meth:`~neptune.experiments.Experiment.get_tags`
+Fetching Data From Neptune
+--------------------------
+
+In addition to uploading experiment data to Neptune, neptune-client can be used to fetch data directly from Neptune for further processing, such as for custom analysis.
+
+`Study the example <https://ui.neptune.ai/USERNAME/example-project/n/Experiments-analysis-with-Query-API-and-Seaborn-31510158-04e2-47a5-a823-1cd97a0d8fcd/91350522-2b98-482d-bc14-a6ff5c061b6b>`_.
+
+On the Project Level
```

Comment:
maybe these heading should be renamed. This exact formulation is not very informative.